doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -331,6 +331,21 @@ You can use the following optional arguments:
 * ``-m`` or ``--matter-path`` to provide a path to a different Matter SDK location than the default one.
   The command will use this path to read the required ZAP tool version and use the generation script from it.
 
+.. _ug_matter_gs_tools_matter_quickstart:
+
+Matter Quick Start app
+**********************
+
+The Matter Quick Start app is a GUI tool for setting up and configuring Matter accessory devices for different smart home ecosystems.
```

Comment:
The current sentence is quite generic and I don't feel like it gets exactly to the point of what we wanted to achieve. Can we try to write something that highlights two major goals of this tool?
1. Enabling fast and easy evaluation of Matter samples without a need of compiling them or setting the development environment thanks to pre-compiled hex files.
2. Providing step-by-step instructions to use the Matter devices with commercial ecosystems (the second is kind of already covered in existing text).